crates/forge_analyzer/src/definitions.rs

```diff
@@ -1193,16 +1203,19 @@ impl<'cx> FunctionAnalyzer<'cx> {
             }
         }
         if let Expr::Member(MemberExpr { obj, prop, .. }) = expr {
-            if let Some(class) = self.resolve_class_prop(obj) {
-                if let MemberProp::Ident(ident) = prop {
-                    if let Some((_, def_constructor)) = class
-                        .pub_members
-                        .iter()
-                        .find(|(name, defid)| name == &ident.sym)
+            if let MemberProp::Ident(ident) = prop {
+                if let Some(class) = self.resolve_class_prop(obj) {
+                    if let Some((_, def_constructor)) =
+                        class.pub_members.iter().find(|(name, defid)| {
+                            return name == &ident.sym;
+                        })
                     {
                         let var = self.body.get_or_insert_global(*def_constructor);
                         return Operand::with_var(var);
                     }
+                } else if let Some(def_id_method) = self.resolve_object_prop(obj, prop) {
```

Comment:
`resolve_obj_prop` should just take an `Ident` as the `prop`, since you already destructure the `MemberProp` here.

Reply:
Fixed. I used the function in the res instead. Thank you